website/package.json

```diff
@@ -11,5 +11,9 @@
     "fs-extra": "^9.0.1",
     "react": "^16.8.4"
   },
-  "dependencies": {}
+  "dependenciesMeta": {
+    "docusaurus": {
+      "unplugged": true
```

Comment:
Docusaurus requires write access to `node_modules`, so we cannot use PnP and should unplug it. (https://github.com/yarnpkg/berry/issues/818#issuecomment-579185574)

Reply:
If `docusaurus` is "unplugged" does this mean that it and all of its dependencies are installed into `node_modules`?

Reply:
@kevinbarabash AFAIK, only the dependency itself is unplugged. Its dependencies remain PnP-ed.

Reply:
That's pretty cool.